azure-pipelines.yml

```diff
@@ -0,0 +1,116 @@
+trigger:
+  # start a new build for every push
+  batch: False
+  branches:
+    include:
+      - master
+      - maintenance/*
+
+jobs:
+- job: Windows
+  pool:
+    vmIMage: 'VS2017-Win2016'
+  variables:
+      OPENBLAS_32: https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/openblas-5f998ef_gcc7_1_0_win32.zip
+      OPENBLAS_64: https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/openblas-5f998ef_gcc7_1_0_win64.zip
```

Comment:
I guess these still need to be bumped to OpenBLAS 0.3.4?

Reply:
but okay to merge first and upgrade later, if that's what you prefer @tylerjereddy

Reply:
I was thinking I'd bump the version after 1.2.0 release, and sync with the scipy-wheels repo